core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/dot-edit-page-routing.module.ts

```diff
@@ -20,38 +23,49 @@ const dotEditPage: Routes = [
                 pathMatch: 'full'
             },
             {
+                path: 'content',
                 loadChildren: () =>
                     import('@portlets/dot-edit-page/content/dot-edit-content.module').then(
                         (m) => m.DotEditContentModule
-                    ),
-                path: 'content'
+                    )
             },
             {
+                path: 'layout',
                 loadChildren: () =>
                     import('@portlets/dot-edit-page/layout/dot-edit-layout.module').then(
                         (m) => m.DotEditLayoutModule
                     ),
-                canDeactivate: [LayoutEditorCanDeactivateGuardService],
-                path: 'layout'
+                canDeactivate: [LayoutEditorCanDeactivateGuardService]
             },
             {
+                path: 'rules/:pageId',
                 loadChildren: () =>
-                    import('@portlets/dot-rules/dot-rules.module').then((m) => m.DotRulesModule),
-                path: 'rules/:pageId'
+                    import('@portlets/dot-rules/dot-rules.module').then((m) => m.DotRulesModule)
+            },
+
+            {
+                path: ':pageId/experiments',
+                loadChildren: async () =>
+                    (
+                        await import(
+                            '../dot-experiments/experiments-shell/dot-experiments-shell.module'
+                        )
+                    ).DotExperimentsShellModule
             }
         ]
     },
     {
+        path: 'layout/template/:id/:tabName',
         loadChildren: () =>
             import(
                 '@portlets/dot-edit-page/layout/components/dot-template-additional-actions/dot-template-additional-actions.module'
-            ).then((m) => m.DotTemplateAdditionalActionsModule),
-        path: 'layout/template/:id/:tabName'
+            ).then((m) => m.DotTemplateAdditionalActionsModule)
     }
 ];
 
 @NgModule({
     exports: [RouterModule],
-    imports: [RouterModule.forChild(dotEditPage)]
+    imports: [RouterModule.forChild(dotEditPage)],
+    providers: [DotExperimentsStore]
```

Comment:
Shouldn't this store `DotExperimentsStore` be declared only in `DotExperimentsShellModule` instead of in `DotEditPageRoutingModule`?